content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print_init-defaults.md

```diff
@@ -0,0 +1,79 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+Exibe a configuração de inicialização padrão, que pode ser usada para 'kubeadm init'
+
+### Sinopse
+
+Este comando exibe os objetos como a configuração de inicialização padrão que é usada para 'kubeadm init'.
+
+Observe que os valores confidenciais, como os campos do Token Bootstrap, são substituídos por valores de espaço reservado como "abcdef.0123456789abcdef", a fim de passar na validação, mas não executar o cálculo real para criar um token.
+
+```
+kubeadm config print init-defaults [flags]
+```
+
+### Opções
+
+   <table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--component-configs strings</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Uma lista separada por vírgulas dos objetos da API de configuração exibirá os valores padrão. Valores disponíveis: [KubeProxyConfiguration KubeletConfiguration]. Se essa flag não estiver definida, nenhuma configuração de componente será impressa.</p></td>
```

Comment:
This sentence can be rethought. It's a bit hard for me to parse through it and understand what meaning it was trying to deliver.